app/src/main/java/com/example/ikeahjalpare/service/FurnitureService.kt

```diff
@@ -9,27 +9,28 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class FurnitureService {
-    fun fetchFurnitures() : MutableLiveData<ArrayList<Furniture>> {
+    fun fetchFurniture(): MutableLiveData<ArrayList<Furniture>> {
         //MVD is defined
-        var _furnitures = MutableLiveData<ArrayList<Furniture>>()
+        var furniture = MutableLiveData<ArrayList<Furniture>>()
```

Comment:
the `_camelCase` naming convention is intended for private variables on the class and this variable is a local variable to the `fetchFurniture` function so it should be using `camelCase`

Reply:
Agreed.